adaptive_breakpoints/example/lib/main.dart

```diff
@@ -64,3 +69,190 @@ class AdaptiveBreakpointsExample extends StatelessWidget {
     );
   }
 }
+
+/// [AdaptiveContainer] lets you create a [Container] with adaptive constraints.
+///
+/// The AdaptiveContainer does everything a normal container does but with
+/// adaptive constraints. For more information go to one of the links below.
+///
+/// https://api.flutter.dev/flutter/widgets/Container-class.html
+/// https://material.io/design/layout/responsive-layout-grid.html#breakpoints
+///
+/// This class is useful whenever you want a container to only be active in
+/// certain [AdaptiveWindowType].
+class _AdaptiveContainer extends StatelessWidget {
```

Comment:
So just so I understand this correctly.

The `AdaptiveContainer` was moved from the breakpoints package to another package, and so this breakpoints demo just makes its own private one? Instead could you add a dependency to the `AdaptiveContainer` to this demo?

Reply:
If its not easy to add as a dependency, then you can probably remove most/all of the documentation on the private one and update the comment on it to say that its the same as the public one. If that makes sense?

Reply:
Actually, what I recommend is leave the breakpoints package alone in this PR, and then once the `Adaptive components` package is published, update the breakpoints package to remove the adaptivecontainer and have the examples simplified perhaps?

Reply:
I thought of this but one of the problems with this is that the adaptive components package imports breakpoints which mean it imports the container within that package. This is the reason the adaptive component's container doesn't have the r at the end to differentiate between the two objects. 

To overcome this I need to remove the container from breakpoints. Release 0.0.3 and then make sure that components use that version in order to correct the name.

I don't want breakpoints to be dependent on components. It seems like an unnecessary import.

Reply:
Gotcha,

Perhaps this should still be split up then:

1. Update breakpoints to no longer expose `AdaptiveContainer`
2. Create components package with `AdaptiveContainer`
3. (Optional) Have just the breakpoints example pubspec depend on components (would this break the circular dependency?)